Assets/LeapMotionModules/ElementRenderer/Scripts/Renderers/LeapGuiBakedRenderer.cs

```diff
@@ -0,0 +1,159 @@
+    }
+  }
+
+  public override void OnUpdateRenderer() {
+    base.OnUpdateRenderer();
+
+    if (gui.space is LeapGuiRectSpace) {
+      using (new ProfilerSample("Build Material Data")) {
+        _rect_elementPositions.Clear();
+        foreach (var element in gui.elements) {
+          var guiSpace = transform.InverseTransformPoint(element.transform.position);
+          _rect_elementPositions.Add(guiSpace);
+        }
+      }
+
+      using (new ProfilerSample("Upload Material Data")) {
+        _material.SetVectorArray(RECT_POSITIONS, _rect_elementPositions);
+      }
+    } else if (gui.space is LeapGuiRadialSpace) {
+      var radialSpace = gui.space as LeapGuiRadialSpace;
+
+      using (new ProfilerSample("Build Material Data")) {
+        _curved_elementParameters.Clear();
+        foreach (var element in gui.elements) {
+          var t = radialSpace.GetTransformer(element.anchor) as LeapGuiRadialSpace.IRadialTransformer;
+          _curved_elementParameters.Add(t.GetVectorRepresentation(element));
+        }
+      }
+
+      using (new ProfilerSample("Upload Material Data")) {
+        _material.SetFloat(LeapGuiRadialSpace.RADIUS_PROPERTY, radialSpace.radius);
+        _material.SetVectorArray(CURVED_PARAMETERS, _curved_elementParameters);
+      }
+    }
+
+    using (new ProfilerSample("Draw Meshes")) {
+      foreach (var mesh in _meshes) {
+        Graphics.DrawMesh(mesh, gui.transform.localToWorldMatrix, _material, _layer);
+      }
+    }
+  }
+
+  protected override void setupForBuilding() {
+    if (_shader == null) {
+      _shader = Shader.Find("LeapGui/Defaults/Baked");
+    }
+
+    base.setupForBuilding();
+
+    switch (_motionType) {
+      case MotionType.Translation:
+        _material.EnableKeyword(LeapGui.FEATURE_MOVEMENT_TRANSLATION);
+        break;
+      case MotionType.Full:
+        _material.EnableKeyword(LeapGui.FEATURE_MOVEMENT_FULL);
+        break;
+    }
+
+    if (_motionType != MotionType.None) {
+      if (gui.space is LeapGuiCylindricalSpace) {
+        _material.EnableKeyword(LeapGuiCylindricalSpace.FEATURE_NAME);
+      } else if (gui.space is LeapGuiSphericalSpace) {
+        _material.EnableKeyword(LeapGuiSphericalSpace.FEATURE_NAME);
+      }
+    }
+  }
+
+  protected override void buildTopology(LeapGuiMeshData meshData) {
+    //If the next element is going to put us over the limit, finish the current mesh
+    //and start a new one.
+    if (_verts.Count + meshData.mesh.vertexCount > MeshUtil.MAX_VERT_COUNT) {
+      finishMesh();
+      beginMesh();
+    }
+
+    base.buildTopology(meshData);
+  }
+
+  protected override void postProcessMesh() {
+    base.postProcessMesh();
+
+    //For the baked renderer, the mesh really never accurately represents it's visual position 
+    //or size, so just disable culling entirely by making the bound gigantic.
+    _currMesh.bounds = new Bounds(Vector3.zero, Vector3.one * 100000);
```

Comment:
We could probably have some sort of way to have the user define a bounding box if it becomes a problem, kinda like skinned mesh renderers, which have a similar problem.  We can add that in another PR if it ends up being an issue.